src/Akeneo/Connectivity/Connection/front/src/model/app.ts

```diff
@@ -13,7 +13,21 @@ export type App = {
     connected: boolean;
 };
 
+export interface TestApp {
```

Comment:
```suggestion
export interface TestApp {
```

Why an interface if the other types are `export type` ?

Reply:
I usually `interface` syntax for new declaration
I can convert to `interface` other types. Shall I ?

Reply:
> I usually interface syntax for new declaration
What's the reason ?

Reply:
It was my understanding to [favor interfaces over types](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces)
With that being said, I maybe wrong since both have virtually same features. I would love to know if the team have preferences (or rules to enforce) about this topic

Reply:
There is stuff you can't do with interfaces, like `type ConnectionCode = string | null;`

So, when to typehinting precisely a scalar (when you really need it, like a VO in the php), you start mixing interfaces & types for no good reasons.

Also, the interface syntax allows to add properties to existing interfaces, which IMHO, is not a good thing when typehinting a model.